src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Linux/LinuxUtilizationParser.cs

```diff
@@ -4,14 +4,15 @@
 using System;
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
-using Microsoft.Extensions.ObjectPool;
 using Microsoft.Shared.Diagnostics;
 using Microsoft.Shared.Pools;
 
 namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring.Linux;
 
 /// <summary>
 /// Parses Linux files to retrieve resource utilization data.
+/// This class is not thread safe.
+/// When the same instance is called by multiple threads it may return corrupted data.
```

Comment:
Suggest moving this as `<remarks>`. Intellisense shows those, making the discoverability easier

Reply:
Oh, it should be remarks. Thanks for spotting. :-)